### ISSUE TITLE
Ability to add alt text to images 

### ISSUE THREAD

**Issue by falakamin31** [codeisscience/journal-policy-tracker-frontend#146]
**Description of bug**

The images are having empty alt text, such as (alt= ' '). When images fail to load, and having no alt text, it casts bad user experience.
Users with visual impairments who use screen readers and Web crawlers are unable to understand anything about images

**Steps to reproduce the behavior:**

1. Slow internet connection 
2. To see this behavior, explicitly make your network slow down. Go to chrome Developers tool, open Network Tab, select a slow speed network
3. Use a screen reader or Chrome screen reader extension

**Expected behavior**
The alternative text (alt) should be visible to visually impaired users or screen readers.
When the image fails to load, alt text should also be visible.

**Screenshots**
**Actual behavior**
![logo](https://user-images.githubusercontent.com/91012681/163780690-d6519115-1fef-4be1-b118-3c46779a3d1b.PNG)
![image](https://user-images.githubusercontent.com/91012681/163780827-57f6d3de-2dda-42f8-9dcc-0f3c19a7d24f.PNG)

https://user-images.githubusercontent.com/91012681/163780967-3486232f-af81-4ebf-adaf-bd388c21f4dc.mp4

**Desktop:**

- OS: Windows
- Browser [Firefox Developer Edition, Google Chrome, Brave]

**Comment by falakamin31** [comment]
Please assign this issue to me. I want to work on it.
